src/sql/workbench/browser/designer/media/designer.css

```diff
@@ -46,12 +46,15 @@
 }
 
 .designer-component .issues-container .issue-item .issue-text {
-	flex: 1 1 auto;
```

Comment:
why are you removing this?  please observe the behavior under different circumstances when you resize the ads window and make sure things look good.

Reply:
This is from the demo. If we don't remove the then the more info link is at the right end of the screen instead of right after the issue description

Reply:
![resizing](https://user-images.githubusercontent.com/6411451/168345780-864a6c3d-8596-41f6-8b03-c0d8ed85fcf3.gif)